### PR TITLE
Tests for 'cat' command

### PR DESCRIPTION
**Pull request by anuveyatsu**
- QA tests automation
  - Cat: basic csv
  - Cat: different separators
  - Cat: different encodings (but these, for now, are skipped as not yet implemented)
- Other tests
  - remote excel
  - inconsistent columns

### CODE REVIEW

File: test/cli.test.js [hunk 1]
```diff
@@ -151,3 +151,105 @@ test('validate command - invalid dataset', async t => {
...
+
+test('cat command - basic behaviour', async t => {
+  const path_ = 'test/fixtures/test-data/files/csv/all-schema-types.csv'
+  const results = await runcli('cat', path_)
+  const stdout = results.stdout.split('\n')
+  t.true(stdout[3].includes('│ 1.0  │'))
+})
+
+test('cat command - remote csv file', async t => {
+  const url_ = 'https://raw.githubusercontent.com/frictionlessdata/test-data/master/files/csv/all-schema-types.csv'
```
Comment (Mikanebu) [inline test/cli.test.js:165]: Just suggestion, we can support GitHub files as we did in `data-get`  data cat https://github.com/frictionlessdata/test-data/blob/master/files/csv/all-schema-types.csv?raw=true. But, it is different issue

**Review by Mikanebu — APPROVED** [review]
Looks good!

File: test/cli.test.js [hunk 2]
```diff
@@ -151,3 +151,105 @@ test('validate command - invalid dataset', async t => {
...
+  t.true(stdout[1].includes('number'))
+
+  // Remote files:
+  let url_ = 'https://raw.githubusercontent.com/frictionlessdata/test-data/master/files/csv/separators/semicolon.csv'
+  results = await runcli('cat', url_)
+  stdout = results.stdout.split('\n')
+  t.false(stdout[1].includes(';'))
+  t.true(stdout[1].includes('number'))
+
+  url_ = 'https://raw.githubusercontent.com/frictionlessdata/test-data/master/files/csv/separators/carets.csv'
```
Comment (Mikanebu) [inline test/cli.test.js:218]: So, we are not testing `colon`, `tab`, `pipes`.

File: test/cli.test.js (hunk 1 above)
Comment (anuveyatsu) [reply test/cli.test.js:165]: @Mikanebu I don't think we need, it was kind of experimental for get command

File: test/cli.test.js (hunk 2 above)
Comment (anuveyatsu) [reply test/cli.test.js:218]: @Mikanebu yes, no need to test everything - we test common case which is `;` and extremely rare case which is `^`

**Pull request merged**